mobile/openapi/lib/model/album_response_dto.dart

```diff
@@ -48,6 +49,14 @@ class AlbumResponseDto {
 
   UserResponseDto owner;
 
+  ///
+  /// Please note: This property should have been non-nullable! Since the specification file
+  /// does not include a default value (using the "default:" property), however, the generated
+  /// source code must fall back to having a nullable type.
+  /// Consider adding a "default:" property in the specification file to hide this note.
+  ///
+  DateTime? lastModifiedAssetTimestamp;
+
```

Comment:
Was there a better way to do this to avoid the message?

Reply:
This should be OK, we have other DTOs with properties that have the same warning as well.